code/modules/antagonists/revolutionary/team_revolution.dm

```diff
@@ -0,0 +1,130 @@
+/datum/team/revolution
+	name = "Revolution"
+	antag_datum_type = /datum/antagonist/rev
+	var/max_headrevs = REVOLUTION_MAX_HEADREVS // adminbus is possible
+
+/datum/team/revolution/New()
+	..()
+	update_team_objectives()
+	SSshuttle.registerHostileEnvironment(src)
+
+/datum/team/revolution/Destroy(force, ...)
+	SSticker.mode.rev_team = null
+	SSshuttle.clearHostileEnvironment(src)
+	. = ..()
+
+
+/datum/team/revolution/get_target_excludes()
+	return ..() + get_targetted_head_minds()
```

Comment:
How so?

Reply:
I'm not exactly sure, just something about it makes me nervous
I feel like this is the *exact* situation where some random undocumented """feature""" will cause issues, it just seems like something that would randomly break, more of a comment vs criticism of your code

Reply:
I feel as it would be pretty easy to backtrack from, and in this case, it only changes the objectives for rev teams, nothing for antag teams as a whole